src/components/profile-view/profile-view.jsx

```diff
@@ -0,0 +1,129 @@
+import { useState } from "react";
+import { Button, Card, Col, Form, Row } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import { MovieCard } from "../movie-card/movie-card";
+
+export const ProfileView = ({ movies }) => {
+  const localUser = JSON.parse(localStorage.getItem("user"));
+  const [user] = useState(localUser ? localUser : null);
```

Comment:
No need to create a state variable if you don't intend to modify it inside the scope of the component.

Reply:
fixed